gci/views.py

```diff
@@ -44,8 +44,8 @@ def gci_overview():
         org_logo_file.write(org_logo)
 
     s.append('<link rel="shortcut icon" type="image/png" '
-             'href="static/favicon.png"/>')
-    s.append('<img src="static/org_logo.png" alt="'+org_name+'">')
+             'href="../static/favicon.png"/>')
```

Comment:
After looking at your patch, I think we need to do the logo as a separate PR, and it should wait until we've done some other improvements to the infrastructure.
I've created an issue for it at https://github.com/coala/community/issues/42